### DISCUSSION TITLE
How to adjust vertical q-slider css-height to its maximum boundaries?

### DISCUSSION THREAD

**Discussion by meo1** [quasarframework/quasar#16351] [General - Components / Directives / etc] [answered]
How to extend vertical q-slider height to its maximum boundaries?

![image](https://github.com/quasarframework/quasar/assets/35503088/2dce4786-c8eb-4820-bc1f-9cee9675ab9a)

**Comment by metalsadman** [comment, marked as answer]
I would expect that would be a 2 column row, one for the inputs and one for the qslider with full-height. https://codepen.io/metalsadman/pen/xxmPwdQ?editors=1010

**Reply by meo1** [reply]
Thank you.
